CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Deprecations
 
 ### Additions and Improvements
+- `txpool_besuPendingTransactions`change parameter `limit` to optional parameter [#6708](https://github.com/hyperledger/besu/pull/6708)
```

Comment:
Name the parameter as per documentation https://besu.hyperledger.org/public-networks/reference/api#parameters-92

```suggestion
- `txpool_besuPendingTransactions`change parameter `numResults` to optional parameter [#6708](https://github.com/hyperledger/besu/pull/6708)
```